Composite/Data/Hierarchy/DataAncestorProviders/PageDataAncestorProvider.cs

```diff
@@ -22,7 +23,21 @@ public IData GetParent(IData data)
 
             using (new DataScope(data.DataSourceId.LocaleScope))
             {
-                return PageManager.GetPageById(parentId);
+                var parent =  PageManager.GetPageById(parentId);
+                if (parent == null && GlobalSettingsFacade.AllowChildPagesTranslationWithoutParent)

```

Comment:
Nope, it is a parent for a relationship, not an actual tree. While the direct parent does not exist in the relationship, it will be the first ancestor.